smartcontract/service/native/ong/ong.go

```diff
@@ -74,8 +74,8 @@ func OngTransfer(native *native.NativeService) ([]byte, error) {
 	}
 	contract := native.ContextRef.CurrentContext().ContractAddress
 	for _, v := range transfers.States {
-		if v.Value == 0 {
-			continue
+		if v.Value < native.MinOngLimit {
```

Comment:
why need this limitation?

Reply:
operator need to commit state to ontology mainnet. if ong too small, user can attack。

Reply:
please clearify how to avoid the attacker to transfer ong upper than MinOngLimit between two accounts back and forth.

Reply:
This is configurable. 
You can configure it to 1 If you do not want this limitation.
That means there is no this limitation.